### PR TITLE
[15.0][MIG] account_financial_report

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: account_financial_report/static/src/js/action_manager_report.js [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-                    "account_financial_report.client_action",
-                    clientActionOptions
-                );
-            }
-            return this._super.apply(this, arguments);
-        },
-    });
-});
+// Method is available here https://github.com/odoo/odoo/blob/15.0/addons/web/static/src/webclient/actions/action_service.js#L981
+// TO DO: Check for implement this action inherit
```
Comment (JasminSForgeFlow) [inline account_financial_report/static/src/js/action_manager_report.js:2]: Hi @Tardo,

Can you please help or hint here with Inherit report action?

**Review by yankinmax — APPROVED** [review]

**Review by Shide — APPROVED** [review]

**Comment by rafaelbn** [comment]
Hello, @JasminSForgeFlow @AaronHForgeFlow @HaraldPanten should this PR take care of:

- [ ] https://github.com/OCA/account-financial-reporting/pull/864
- [ ] https://github.com/OCA/account-financial-reporting/pull/866 (I confirm this BUG)
- [ ] https://github.com/OCA/account-financial-reporting/pull/850

I think @JasminSForgeFlow if possible we should merge v14 fixes and get all fixed in v15 too

What do you think?

Thanks

**Comment by HaraldPanten** [comment]
I agree. We have some pending fixes that might make the module more "stable" for V15.

On the other hand, maybe we should support/approve the fixes as authors are active users, and probably they're close to be merged.

**Review by rafaelbn — APPROVED** [review]
OK 👍 as said.

**Comment by rafaelbn** [comment]
Hello @JasminSForgeFlow could you please check pre-commit? https://github.com/OCA/account-financial-reporting/runs/5821935697?check_suite_focus=true

@AaronHForgeFlow can you help?

**Comment by JasminSForgeFlow** [comment]
> Hello @JasminSForgeFlow could you please check pre-commit? https://github.com/OCA/account-financial-reporting/runs/5821935697?check_suite_focus=true
> 
> @AaronHForgeFlow can you help?

Updated.

Thanks

**Pull request merged**